graphql_compiler/compiler/expressions.py

```diff
@@ -508,6 +515,82 @@ def to_gremlin(self):
         raise AssertionError(u'ContextFieldExistence.to_gremlin() was called: {}'.format(self))
 
 
+def _validate_operator_name(operator, supported_operators):
+    """Ensure the named operator is valid and supported."""
+    if not isinstance(operator, six.text_type):
+        raise TypeError(u'Expected unicode operator, got: {} {}'.format(
```

Comment:
you are testing for text_type, the error is "expected unicode operator"

Reply:
`text_type` is `unicode` in python 2, and `str` in python 3 since `str` there is a list of unicode code points. Any recommendations for how to rephrase the error to make this more clear?

Reply:
Expected operator as unicode string?

Reply:
Thanks, will change to that before merging.